tunnel/wstuncli.go

```diff
@@ -103,6 +106,7 @@ func NewWSTunnelClient(args []string) *WSTunnelClient {
 	var statf *string = cliFlag.String("statusfile", "", "path for status file")
 	var proxy *string = cliFlag.String("proxy", "",
 		"use HTTPS proxy http://user:pass@hostname:port")
+	var cliport *string = cliFlag.String("clientports", "", "comma separated list of client listening ports")
```

Comment:
Is a single comma separated list going to be sufficient? What is a typical use case? Just a few ports? Do we need to support specifying a range and we can still accept comma separated values  (i.e) `--client-ports 8000..8100,8300..8400,8500,8505`?

Reply:
Most likely it will be 1 port as they will be nat/pat, but I like the range idea it will give me more golang experience so I will attempt it.